libkpod/remove.go

```diff
@@ -0,0 +1,62 @@
+package libkpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/sirupsen/logrus"
+)
+
+// RemoveContainers removes the given containers. If the container is running, the container
+// should be force removed.
+func (c *ContainerServer) RemoveContainers(containers []string, force bool) string {
+	var ret string
+	for _, container := range containers {
+		id, err := c.remove(container, force)
+		if err != nil {
+			ret = append(ret, err.Error(), "\n")
```

Comment:
Should we return a list of containers that failed and the errors that were returned?